yoga/Yoga.c

```diff
@@ -423,7 +423,15 @@ inline float YGNodeStyleGetFlexGrow(const YGNodeRef node) {
   return 0.0f;
 }
 
-inline float YGNodeStyleGetFlexShrink(const YGNodeRef node) {
+float YGNodeStyleGetFlexGrow(const YGNodeRef node) {
+  return YGFloatIsUndefined(node->style.flexGrow) ? 0.0f : node->style.flexGrow;
```

Comment:
Instead of 0.0f, can we have the reference the defaults instead for clarity and to not introduce bugs?